x/stakeibc/keeper/icqcallbacks_delegator_shares_test.go

```diff
@@ -272,79 +219,24 @@ func (s *KeeperTestSuite) TestDelegatorSharesCallback_Retry_DelegationChange() {
 func (s *KeeperTestSuite) TestDelegatorSharesCallback_Retry_DelegationICAInProgress() {
 	tc := s.SetupDelegatorSharesICQCallback()
 
-	// Add a deposit record that makes it appear as if a delegation ICA is in progress
-	s.App.RecordsKeeper.SetDepositRecord(s.Ctx, recordstypes.DepositRecord{
-		Id:         100,
-		HostZoneId: HostChainId,
-		Status:     recordstypes.DepositRecord_DELEGATION_IN_PROGRESS,
-	})
+	// Update the validator's delegation change ICA counter to show a change is in progress
+	initialHostZone := tc.hostZone
+	initialHostZone.Validators[tc.valIndexQueried].DelegationChangesInProgress = 1
+	s.App.StakeibcKeeper.SetHostZone(s.Ctx, initialHostZone)
 
 	// Callback
 	err := stakeibckeeper.DelegatorSharesCallback(s.App.StakeibcKeeper, s.Ctx, tc.validArgs.callbackArgs, tc.validArgs.query)
 	s.Require().NoError(err, "no error expected during delegator shares callback")
 
 	// Confirm the validator's delegation was not modified
-	hostZone, found := s.App.StakeibcKeeper.GetHostZone(s.Ctx, tc.hostZone.ChainId)
+	actualHostZone, found := s.App.StakeibcKeeper.GetHostZone(s.Ctx, HostChainId)
 	s.Require().True(found, "host zone found")
 
-	initialDelegation := hostZone.Validators[tc.valIndexQueried].Delegation
-	s.Require().Equal(initialDelegation.Int64(), hostZone.Validators[tc.valIndexQueried].Delegation.Int64(), "validator delegation")
+	initialDelegation := actualHostZone.Validators[tc.valIndexQueried].Delegation
```

Comment:
I think you want the initialHostZone here, you are checking the same literal value against itself which will always be true on next line.

Reply:
nice catch! done